services/src/main/java/org/keycloak/services/resources/admin/UsersResource.java

```diff
@@ -453,6 +454,9 @@ public UserProfileResource userProfile() {
                             ? ModelToRepresentation.toBriefRepresentation(user)
                             : ModelToRepresentation.toRepresentation(session, realm, user);
                     userRep.setAccess(usersEvaluator.getAccess(user));
+                    if (session.getProvider(BruteForceProtector.class).isTemporarilyDisabled(session, realm, user)) {
```

Comment:
One thing that it should be simple to include and help to not impact runtime performance when brute force is disabled is to make sure we only run this code if the realm has bruteforce enabled. 

You can infer that by looking at `RealmModel.isBruteForceProtected`.

Reply:
I've added a check for this.